libcontainer/system/sysconfig.go

```diff
@@ -10,3 +10,7 @@ import "C"
 func GetClockTicks() int {
 	return int(C.sysconf(C._SC_CLK_TCK))
 }
+
+func GetLongBit() int {
+	return int(C.sysconf(C._SC_LONG_BIT))
```

Comment:
It might be sufficient to add something like this: (untested)
```go
/*
#ifndef _SC_LONG_BIT
#define _SC_LONG_BIT (8*sizeof(long))
#endif
*/
import "C"
```